crates/re_renderer/src/renderer/rectangles.rs

```diff
@@ -113,6 +122,17 @@ impl ColormappedTexture {
             gamma: 1.0,
             multiply_rgb_with_alpha: true,
             color_mapper: None,
+            encoding: None,
+        }
+    }
+
+    pub fn width_height(&self) -> [u32; 2] {
+        match self.encoding {
+            Some(TextureEncoding::Nv12) => {
+                let [width, height] = self.texture.width_height();
+                [width, (height as f64 / 1.5) as u32]
```

Comment:
nit: as below, let's not do the float roundtrip and instead do
```suggestion
                [width, height * 2 / 3]
```